images/istio/tests/main.tf

```diff
@@ -21,3 +22,8 @@ data "oci_exec_test" "pilot-version" {
   digest = var.digests.pilot
   script = "docker run --rm $IMAGE_NAME --version"
 }
+
+data "oci_exec_test" "operator-version" {
```

Comment:
Is there a Helm chart or something we can use to see that this runs in a real cluster?

Reply:
Strangely enough there is one but its deprecated. I found the most adopted one and added it in here.